src/main/resources/static_config/static_roles.yml

```diff
@@ -132,6 +132,7 @@ logstash:
   index_permissions:
     - index_patterns:
         - "logstash-*"
+        - "ecs-logstash-*"
```

Comment:
I am wondering whether ecs-logstash-* is a industry-standard index that is created/if it isn't then maybe this should be added via another role?

Reply:
Just re-read the issue and I don't see any harm in adding this, if it is supported by logstash-output-opensearch plugin

Reply:
Thanks @derek-ho and @cwperks for reviewing this. Is it OK to resolve this thread?

Reply:
Yes please, go ahead. I can't resolve it from my end unfortunately.